### PR TITLE
CSRF documentation fix

### PR DESCRIPTION
**Pull request by bobalazek**

### CODE REVIEW

File: doc/providers/csrf.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ Service Provider are protected against CSRF by default.
 
 You can also use the CSRF protection even without using the Symfony Form
 component. If, for example, you're doing a DELETE action, you can check the
-CSRF token::
+CSRF token:
 
 .. code-block:: php
```
Comment (fabpot) [inline doc/providers/csrf.rst]: You should remove this line instead as PHP is the default.

Comment (bobalazek) [inline doc/providers/csrf.rst]: Sure? There's a lot other instances that also have the same php code-block.

Comment (fabpot) [inline doc/providers/csrf.rst]: Indeed. If you are interested, can you make a pull request on Silex 1.3 with all those changed? Thanks.

Comment (bobalazek) [inline doc/providers/csrf.rst]: Ok yes. Will fix it later today. I'll probably go through the 2.0 documentation first (I've seen a few typos and compatibility changes there). So I'll close this PR for now and create a new one later with all the changes.

Comment (fabpot) [inline doc/providers/csrf.rst]: You should start with 1.3 first as the 1.3 branch is then merged in master/2.0.

Comment (bobalazek) [inline doc/providers/csrf.rst]: Ok

**Pull request closed without merging**